data/hai jobs.txt

```diff
@@ -141,14 +143,15 @@ mission "Wealthy Hai [2]"
 	to offer
 		random < 30
 	on offer
+		require "Luxury Accommodations"
 		require "Quantum Keystone"
```

Comment:
If the Wealthy Hai jobs would now require both a Quantum Keystone and luxury accommodations, perhaps also add a job or jobs that require only one or the other (e.g. a job for only keystones and a job for only accommodations in Hai space). Otherwise, it'll be more difficult for a player to make the connection between having a keystone and having these jobs offer.

Reply:
"These superstitious Hai are looking for transport to <planet>, but believe that in order for their journey to be successful they must travel exclusively on a ship with the proper crystal on it."

Reply:
Hmm, that's true. I'll see what I can do.

Reply:
I've adjusted the description of the Quantum Keystone instead, to hint at it being a piece of puzzle to unlocking the wealthy Hai passenger jobs.

Reply:
I think that's good for now, but I would rather have Hai jobs that only require one of each trinket at some point.